docs/standard/parallel-programming/how-to-write-a-simple-parallel-foreach-loop.md

```diff
@@ -37,14 +39,14 @@ You can also use Parallel LINQ (PLINQ) to parallelize processing of <xref:System
 
 ## Compile and run the code
```

Comment:
This needs some major work. This can be done as a .NET Framework console application or a .NET Core console application. It can also be done with or without Visual Studio.  Above all, it's important that examples work on both .NET Framework and .NET Core.
- In Visual Studio, there are console application templates for VB and C# for both Desktop Development and .NET Core.
- From the command line, you can use either .NET Core and its CLI tools (`dotnet new console` and `dotnet new console -lang vb`), or you can simply create the file and use the command-line compiler for a .NET Framework application.
- For a .NET Core project, you need to reference the NuGet package **System.Drawing.Common**.  In Visual Studio, you can use NuGet Package Manager to install it. You can also add the following to the *.csproj or *.vbproj file:
```
  <ItemGroup>
    <PackageReference Include="System.Drawing.Common" Version="4.5.1" />
  </ItemGroup>
```